src/test/java/com/puppycrawl/tools/checkstyle/gui/ParseTreeTablePresentationTest.java

```diff
@@ -258,46 +263,50 @@ public void testGetValueAtDetailNode() {
         final String text = (String) parseTree.getValueAt(child, 4);
         final String expectedText = "JAVADOC";
 
-        Assert.assertNull("Tree model must be null", treeModel);
-        Assert.assertEquals("Invalid type", "JAVADOC", type);
-        Assert.assertEquals("Invalid line", 3, line);
-        Assert.assertEquals("Invalid column", 3, column);
-        Assert.assertEquals("Invalid text", expectedText, text);
+        assertNull(treeModel, "Tree model must be null");
+        assertEquals("JAVADOC", type, "Invalid type");
+        assertEquals(3, line, "Invalid line");
+        assertEquals(3, column, "Invalid column");
+        assertEquals(expectedText, text, "Invalid text");
 
         try {
             parseTree.getValueAt(child, parseTree.getColumnCount());
-            Assert.fail("IllegalStateException expected");
+            fail("IllegalStateException expected");
         }
         catch (IllegalStateException ex) {
-            Assert.assertEquals("Invalid error message", "Unknown column", ex.getMessage());
+            assertEquals("Unknown column", ex.getMessage(), "Invalid error message");
         }
     }
 
     @Test
     public void testColumnMethods() {
         final ParseTreeTablePresentation parseTree = new ParseTreeTablePresentation(null);
-        Assert.assertSame("Invalid type", ParseTreeTableModel.class, parseTree.getColumnClass(0));
-        Assert.assertSame("Invalid type", String.class, parseTree.getColumnClass(1));
-        Assert.assertSame("Invalid type", Integer.class, parseTree.getColumnClass(2));
-        Assert.assertSame("Invalid type", Integer.class, parseTree.getColumnClass(3));
-        Assert.assertSame("Invalid type", String.class, parseTree.getColumnClass(4));
+        assertSame(ParseTreeTableModel.class, parseTree.getColumnClass(0), "Invalid type");
+        assertSame(String.class, parseTree.getColumnClass(1), "Invalid type");
+        assertSame(Integer.class, parseTree.getColumnClass(2), "Invalid type");
+        assertSame(Integer.class, parseTree.getColumnClass(3), "Invalid type");
+        assertSame(String.class, parseTree.getColumnClass(4), "Invalid type");
 
         try {
             parseTree.getColumnClass(parseTree.getColumnCount());
-            Assert.fail("IllegalStateException expected");
+            fail("IllegalStateException expected");
         }
         catch (IllegalStateException ex) {
-            Assert.assertEquals("Invalid error message", "Unknown column", ex.getMessage());
+            assertEquals("Unknown column", ex.getMessage(), "Invalid error message");
         }
 
-        Assert.assertFalse("Invalid cell editable status", parseTree.isCellEditable(1));
+        assertFalse(parseTree.isCellEditable(1), "Invalid cell editable status");
+    }
 
```

Comment:
This test was split into two to satisfy the PMD rule "no more than 11 asserts per method"